bundles/org.openhab.binding.volvooncall/src/main/resources/OH-INF/thing/vehicle.xml

```diff
@@ -385,18 +389,49 @@
 	<channel-type id="batteryLevel">
 		<item-type>Number:Dimensionless</item-type>
 		<label>Battery Level</label>
-		<description>Indicates the level of power in the battery (in case of PHEV / Twin Engine)</description>
+		<description>Indicates the level of power in the battery, or unknown in situations the API is misleading (in case of
+			PHEV / Twin Engine)</description>
+		<category>batterylevel</category>
+		<state pattern="%d %unit%" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="batteryLevelRaw">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Battery Level Raw</label>
+		<description>Indicates the level of power in the battery taken straight from the API, which can be misleading (in case
+			of PHEV / Twin Engine)</description>
 		<category>batterylevel</category>
 		<state pattern="%d %unit%" readOnly="true"></state>
 	</channel-type>
 
 	<channel-type id="chargeStatus">
 		<item-type>String</item-type>
-		<label>Charging status</label>
+		<label>Charging status description</label>
```

Comment:
Please keep labels short (max 20-25 chars) and always use capitalization. (Would be great if you could check all other labels in this binding as well for this.)

```suggestion
		<label>Charging Status Desc.</label>
```